packages/react-components/react-alert/src/components/Alert/useAlertStyles.ts

```diff
@@ -1,34 +1,78 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { tokens } from '@fluentui/react-theme';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+
 import type { AlertSlots, AlertState } from './Alert.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const alertClassName = 'fui-Alert';
```

Comment:
```suggestion
```
`alertClassNames.root` should be used instead

Reply:
For context https://github.com/microsoft/fluentui/issues/22132

Reply:
Yup, I am already using `alertClassNames.root`. @khmakoto mentioned that className was being exported for compatibility reasons. Will remove if there are no build errors.

Reply:
Removing this export causes the isConformant check in the tests to fail. Added that line back.